src/test/resources/jsonld/HT017536715

```diff
@@ -0,0 +1,140 @@
+    "id" : "http://lobid.org/resources/HT006816401#!",
+    "label" : "Jahrbuch / Mülheim an der Ruhr"
+  } ],
+  "hbzId" : "HT017536715",
+  "inCollection" : [ {
+    "id" : "http://lobid.org/resources/HT014176012#!",
+    "type" : [ "Collection" ],
+    "label" : "Nordrhein-Westfälische Bibliographie (NWBib)"
+  } ],
+  "subjectAltLabel" : [ "Evangelische Kirche im Rheinland. Evangelischer Kirchenkreis An der Ruhr", "Mülheim (Ruhr)", "Kirchenkreis An der Ruhr", "Mülheim, Ruhr", "Mülheim an der Ruhr. Oberstadtdirektor", "Mülheim an der Ruhr. Stadtverwaltung", "Evangelischer Kirchenkreis An der Ruhr", "Mülheim an der Ruhr. Hauptamt", "Mülheim a. d. Ruhr" ],
+  "otherTitleInformation" : [ "Luthers Erben an der Ruhr" ],
+  "publication" : [ {
+    "type" : [ "PublicationEvent" ],
+    "location" : "Mülheim, Ruhr",
+    "startDate" : "2012"
+  } ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT017536715",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "describedBy" : {
+    "id" : "http://lobid.org/resources/HT017536715",
+    "type" : [ "BibliographicDescription" ],
+    "modifiedBy" : {
+      "id" : "http://lobid.org/organisations/DE-61#!",
+      "label" : "lobid Organisation"
+    },
+    "dateCreated" : "20130206",
+    "dateModified" : "20190410",
+    "inDataset" : {
+      "id" : "http://lobid.org/resources/dataset#!",
+      "label" : "lobid-resources – Der hbz-Verbundkatalog als Linked Open Data"
+    },
+    "resultOf" : {
+      "type" : [ "CreateAction" ],
+      "endTime" : "0001-01-01T00:00:00",
+      "instrument" : {
+        "id" : "https://github.com/hbz/lobid-resources",
+        "type" : [ "SoftwareApplication" ],
+        "label" : "Software lobid-resources"
+      },
+      "object" : {
+        "id" : "http://lobid.org/hbz01/HT017536715",
+        "type" : [ "DataFeedItem" ],
+        "inDataset" : {
+          "id" : "https://datahub.io/dataset/hbz_unioncatalog",
+          "label" : "hbz_unioncatalog"
+        },
+        "label" : "hbz-Ressource HT017536715 im Exportformat MAB2-XML"
+      }
+    },
+    "license" : [ {
+      "id" : "http://creativecommons.org/publicdomain/zero/1.0",
+      "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
+    } ],
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-Bm3#!",
+      "label" : "lobid Organisation"
+    },
+    "label" : "Webseite der hbz-Ressource HT017536715"
+  },
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "id" : "http://lobid.org/resources/HT017536715#!",
+  "spatial" : [ {
+    "focus" : {
+      "id" : "http://www.wikidata.org/entity/Q63078887",
+      "type" : [ "http://www.wikidata.org/entity/Q897303" ]
+    },
+    "id" : "https://nwbib.de/spatial#Q63078887",
+    "type" : [ "Concept" ],
+    "label" : "Kirchenkreis An der Ruhr",
+    "source" : {
+      "id" : "https://nwbib.de/spatial",
+      "label" : "Raumsystematik der Nordrhein-Westfälischen Bibliographie"
+    }
+  } ]
```

Comment:
The generic N35 "Evangelische Kirche. Kirchenkreise" object is gone now as expected but another object is gone too that should still be there: "Mülheim an der Ruhr" (Q2899), see http://lobid.org/resources/HT017536715.json. From the [source](http://lobid.org/hbz01/HT017536715):

```xml
<datafield tag="700" ind1="n" ind2="1">
  <subfield code="a">99</subfield>
  <subfield code="b">Mülheim an der Ruhr</subfield>
</datafield>
<datafield tag="700" ind1="n" ind2="1">
  <subfield code="a">35</subfield>
  <subfield code="b">Kirchenkreis An der Ruhr</subfield>
</datafield>
```

Reply:
Q2899 wasn't yet in the wikidata test's database.
Added it.